### DISCUSSION TITLE
5.0.0: major release for christmas

### DISCUSSION THREAD

**Discussion by mistic100** [mistic100/Photo-Sphere-Viewer#798] [Announcements]
Version 5 of Photo Sphere Viewer is now available. It's been a long work of rewriting the library in TypeScript, splitting in multiple NPM packages, experiment with ESBuild, automate the release process and of course fix a few bugs! Hop in the [migration guide](https://photo-sphere-viewer.js.org/guide/migration.html) to update your application.

If you have any problem with this version, like a bug or an usage question, please open a new issue.

**Comment by ahalp0217** [comment]
now available*

**Reply by ahalp0217** [reply]
Very cool, congrats!
